web_responsive/static/src/css/web_responsive.scss

```diff
@@ -324,19 +344,67 @@ html .o_web_client .o_main .o_main_content {
 .o_content, .modal-content {
     max-width: 100%;
 
+    // Sticky Header & Footer in List View
+    >div {
+        >.table-responsive {
```

Comment:
Can't we just apply it to all `.table-responsive` found? If possible it's good to `avoid > too > strict > structures`. It's usually better to be less strict and just apply to classes inside classes or such. In a future version they could add/remove an element in the chain and nobody would notice this section is not working. Being less strict, that's less possible. It's also a new approach in BS4.

Of course, it's not always possible; I'm not sure if it's the case here...